gammapy/irf/psf_map.py

```diff
@@ -299,6 +299,48 @@ def from_energy_dependent_table_psf(cls, table_psf):
         exposure_map.quantity += data
         return cls(psf_map=psf_map, exposure_map=exposure_map)
 
+    @classmethod
+    def from_gauss(cls, energy, rad, sigma):
+        """Create all -sky PSF map from Gaussian width.
+
+        This is used for testing and examples.
+
+        The width can be the same for all energies
+        or be an array with one value per energy node.
+        It does not depend on position.
+
+        Parameters
+        ----------
+        energy : `~astropy.units.Quantity`
+            Energy (1-dim)
+        rad : `~astropy.units.Quantity` with angle units
+            Offset angle wrt source position (1-dim)
+        sigma : `~astropy.coordinates.Angle`
+            Gaussian width.
+        Returns
+        -------
+        psf_map : `PSFMap`
+            Point spread function map.
+        """  
+
+        # note: it would be straightforward to also have disk shape instead
+        # of gauss
+        tableshape = (energy.shape[0], rad.shape[0])
+        if np.size(sigma) == 1:
```

Comment:
Minor: for 1d arrays we typically use the standard Python `len()` expression...

Reply:
If `sigma` is just a float or an `astropy.Quantity`, `len(sigma)` throws an error. Also  `np.size("0.1deg") = 1`, while` len("0.1deg")=6`.

Reply:
Ah I see, makes sense to keep...!